### ISSUE TITLE
Notify on QR code regenerate

### ISSUE THREAD

**Issue by jordann1** [NightfallGT/Discord-QR-Scam#6]
sometimes if a user scans the qr code and clicks cancel instead of log in the qr code gets invalidated

**Comment by CA1R7** [comment]
@jordann1 Yes cuz user noticed this qr code of logging, This scam for stupid peoples!!
